.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Let's remove this file. I propose to have `.idea/` in your global .gitignore file.